tools/c7n_azure/c7n_azure/resources/vnet.py

```diff
@@ -32,4 +34,51 @@ class resource_type(ArmResourceManager.resource_type):
         service = 'azure.mgmt.network'
         client = 'NetworkManagementClient'
         enum_spec = ('virtual_networks', 'list_all', None)
+        default_report_fields = (
+            'name',
+            'location',
+            'resourceGroup'
+        )
         resource_type = 'Microsoft.Network/virtualNetworks'
+
+@Vnet.filter_registry.register('gw-without-ipsec-policies')
+class IPSecAlgorithmFilter(ValueFilter):
+    """Virtual Networks Gateway Resource IPSec configured
+
+    :example:
+
+    This set of policies will find all Virtual Networks that are not
+      configured with a cryptygraphic algorithm.
+
+    .. code-block:: yaml
+
+        policies:
+          - name: gateway-configured-with-cryptographic-algorithm
+            resource: azure.vnet
+            filters: 
+              -type: configured-with-cryptographic-algorithm,
```

Comment:
needs a space between `-` and `type`, currently results in ci failure on doc example testing as this is interpreted by yaml as a dict key instead of a list.